signedjson/key.py

```diff
@@ -151,7 +151,7 @@ def read_signing_keys(stream):
 
 
 def read_old_signing_keys(stream):
-    # type: (Iterable[str]) -> List[VerifyKey]
+    # type: (Iterable[str]) -> List[VerifyKeyWithExpiry]
```

Comment:
Should we inline these types now? I'm not sure the minimum version we support on this.

Reply:
Probably, though I'm not sure we get anything out of doing so.

I'd like to make that a separate change though, so that this one is just "get our usual linting tools passing".

Reply:
> Probably, though I'm not sure we get anything out of doing so.

It is just slightly nicer, I think.

> I'd like to make that a separate change though, so that this one is just "get our usual linting tools passing".

Separate is fine, I think `pyupgrade` will do it for you automatically.